docker_sample/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rust:1.41.0-stretch as builder
```

Comment:
Not sure about this one but: wouldn't it be better to use the `1` tag of rust image? Actix itself will be maintained to work with stable releases of Rust, so it would be nice if the Dockerfile don't need to be updated in case we raise the minimal required Rust version to a new stable release.